scrapinghub/client.py

```diff
@@ -203,6 +195,17 @@ def _extract_spider_id(self, params):
             raise ValueError('Please use same spider id')
         return spiderid
 
+    def update_tags(self, add=None, remove=None, spidername=None):
+        spidername = spidername or (self.spider.name if self.spider else None)
+        if not spidername:
+            raise ValueError('Please provide spidername')
```

Comment:
Sure, it covers the case when we work with project jobs (`project.jobs`), that's a trick to have the same logic for project jobs and spider jobs